src/main/scala/org/kidsfirstdrc/dwh/external/ImportClinVar.scala

```diff
@@ -1,47 +1,129 @@
 package org.kidsfirstdrc.dwh.external
 
+import org.apache.spark.sql.expressions.UserDefinedFunction
 import org.apache.spark.sql.functions._
-import org.apache.spark.sql.{Column, DataFrame, SaveMode, SparkSession}
+import org.apache.spark.sql._
+import org.kidsfirstdrc.dwh.utils.EtlJob
 import org.kidsfirstdrc.dwh.utils.SparkUtils._
 import org.kidsfirstdrc.dwh.utils.SparkUtils.columns._
 
-object ImportClinVar extends App {
+import scala.collection.mutable
+
+object ImportClinVar extends App with EtlJob {
 
   implicit val spark: SparkSession = SparkSession.builder
     .config("hive.metastore.client.factory.class", "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory")
     .enableHiveSupport()
     .appName("Import ClinVar").getOrCreate()
 
-  import spark.implicits._
-
   val input = "s3a://kf-strides-variant-parquet-prd/raw/clinvar/clinvar_20201026.vcf.gz"
   val output = "s3a://kf-strides-variant-parquet-prd/public"
 
+  val sourceDF = extract(input)
+  val resultDF = transform(sourceDF)
+  load(resultDF, output)
+
+  override def extract(input: String)(implicit spark: SparkSession): DataFrame = {
+    vcf(input)
+  }
+
+  override def transform(data: DataFrame)(implicit spark: SparkSession): DataFrame = {
+    spark.udf.register("allele_origin", allele_origin_udf)
+
+    val intermediateDf =
+      data
+      .select(
+        chromosome +:
+          start +:
+          end +:
+          name +:
+          reference +:
+          alternate +:
+          (col("INFO_CLNSIG") as "clin_sig") +:
```

Comment:
Are these constants used elsewhere in the app? If so, would be a little bit safer to extract them into constant variables?